app/src/main/java/ar/com/wolox/android/example/utils/UserSession.kt

```diff
@@ -19,4 +19,14 @@ class UserSession @Inject constructor(private val sharedPreferencesManager: Shar
             field = username
             sharedPreferencesManager.store(Extras.UserLogin.USERNAME, username)
         }
+
+    var email: String? = null
+        get() = sharedPreferencesManager["email", null]
+        set(email) {
+            field = email
+            sharedPreferencesManager.store("email", email)
+        }
+
+    val userIsLogged: Boolean?
+        get() = email != null
```

Comment:
I think you should also ask if the email is not empty

Reply:
there is a String function (in kotlin this is also called extension) that you can check: email.isNullOrEmpty(). So you can use that and negate it